packages/chap-lib/src/components/evaluation/ComparionPlotWrapper/ComparionPlotWrapper.module.css

```diff
@@ -9,17 +9,21 @@
     justify-content: space-between;
     gap: 2rem;
     margin-block-end: 10px;
+    /*Struggling to find a good way of doing this. calc(-128px + 100vh) is what the DHIS2 modal uses for height, and 24px is the modal's padding.*/
+    max-height: calc(-128px + 100vh - (24px * 2));
```

Comment:
Yeah, I think the "easiest" solution would be to have `height: 100%` on the parent elements (eg. the `div` rendered in `Modal` (jsx-1532202667), however we can't easily add that to the elements rendered from UI...